content/general/push.textile

```diff
@@ -0,0 +1,102 @@
+---
+title: Push Notifications
+section: general
+index: 55
+api_separator:
+jump_to:
+  Help with:
+    - Delivery push notifications#deliver
+    - Activating and subscribing a device#activate
+    - Managing devices and subscriptions#admin
+    - Platform support#platform-support
+---
+
+Ably can deliver native push notifications to devices using, amongst others, "Apple's Push Notification":https://developer.apple.com/notifications/ service and "Google's Cloud Messaging":https://developers.google.com/cloud-messaging/ service. Native push notifications, unlike Ably's "channel based pub/sub messaging":/realtime/channels, do not require the device to maintain a connection to Ably as the underlying platform or OS is responsible for maintaining its own battery efficient transport to receive push notifications. Therefore, native push notifications are commonly used to display visual notifications to users or launch a background process for an app in a battery efficient manner.
+
+h2(#beta). Beta notice
+
+Ably's push notification service is currently in beta. Please bear in mind:
+
+* Whilst the API we provide now is planned for our GA release, *the API is subject to change*. However, we will endeavour to only make changes to the API where necessary before we transition the service to GA (general availability).
+* The push service, as far as we are aware, is ready for production levels of usage. However, given the beta status, push notifications have not yet been stress tested under real-world production loads. As a result, you should expect some bugs. If you experience any problems, please "get in touch":https://www.ably.io/contact and we'll work as quickly as possible to help identify the problem and fix it.
+* The client libraries currently have varying levels of support for push notifications. Please refer to the "push notification client library download section":/general/push/download for details on how to use push on your devices and servers now.
+* Push notification statistics are currently disabled. Therefore, you are not currently charged for push notifications and and the push notification stats in your "app dashboard":https://support.ably.io/support/solutions/articles/3000030053-how-do-i-access-my-app-dashboard will be zero. Please note that each push notification, when stats are enabled, will "count as one message from a billing perspective":https://support.ably.io/solution/articles/3000053844-how-does-ably-count-messages. We plan to enable push statistics soon, "get in touch":https://www.ably.io/contact if you need statistics now.
+* Documentation is not complete. We'll be continuing to add more documentation over the coming weeks. However if you have any specific questions or recommendations in regards to how we can improve the documentation, please do "get in touch":https://www.ably.io/contact
```

Comment:
`We'll be continuing to add more documentation over the coming weeks.`. We probably won't be, so worth removing?